### PR TITLE
Add 'Seeker' model

### LINKED ISSUE #44: Add 'Seeker' model #40
In order to build our ORM we will add the 'Seeker' class.

The Seeker class extends (OneToOne) the base User class.
The class fields will gather the seeker's 'wanted' apartment data such as:

-  city
- start date 
- min\max rent
- etc.

In addition, the class will be registered to the admin page.

### PR DESCRIPTION
**Pull request by AmitAharoni**
In order to build our ORM we add the `Seeker` class.

The Seeker class extends (OneToOne) the base User class.
The class fields gather the seeker's 'wanted' apartment data such as:

- city
- start date
- min\max rent
- etc.

In addition, the class will be registered to the admin page.

fixes #44

### CODE REVIEW

**Review by DanielMy777 — COMMENTED** [review]
Looks great!
Also cloned it and works like a charm.

File: users/models.py [hunk 1]
```diff
@@ -109,3 +109,17 @@ class Apartment(models.Model):
...
+
+class Seeker(models.Model):
+    base_user = models.OneToOneField(AUTH_USER_MODEL, on_delete=models.CASCADE, primary_key=True)
+    city = models.ForeignKey(City, on_delete=models.RESTRICT, related_name='city_seekers')
+    start_date = models.DateField()
+    min_rent = models.IntegerField()
+    max_rent = models.IntegerField()
+    num_of_roomates = models.IntegerField()
+    num_of_rooms = models.IntegerField()
+    about = models.TextField(blank=True, max_length=300)
```
Comment (tamirho) [inline users/models.py:122]: I think we should not limit the number of characters in the about field.
Besides that, everything looks graet!

**Review by simzacks — APPROVED** [review]

**Review by Micha546 — COMMENTED** [review]
Looks great!

**Review by Omeramsc — APPROVED** [review]
LGTM

**Pull request merged**